### ISSUE TITLE
ReferenceError: Image is not defined

### ISSUE THREAD

**Issue by elibenton** [anotherempty/svelte-brick-gallery#1]
Hello another empty,

Thank you for making this amazing package. I have tried to make something similar many times before but failed — this is a huge help!

I am running into one small issue, which I think is my own fault, but I was wondering if you might be willing to help diagnose the problem.

I am trying to run this program in SvelteKit, fetching images from [Cloudflare Images](https://www.cloudflare.com/products/cloudflare-images/) using an API endpoint. When I run `yarn dev`, I get the following error.

```
ReferenceError: Image is not defined
    at Timeout.eval [as _onTimeout] (/node_modules/svelte-brick-gallery/Image.svelte:34:20)
```

In your [REPL examples](https://svelte.dev/repl/32ab548ff35d4002ac2c3ea35f98812b?version=3.48.0), I see that you have a `load` function and `{#await}` block. My code does not have this, but I thought it wasn't necessary if I used an API endpoint. 

```js
<script context="module">
  export async function load({ fetch }) {
    const url = `/photos.json`
    const res = await fetch(url)

    if (res.ok) {
      return {
        status: res.status,
        props: {
          images: await res.json()
        }
      }
    }

    return {
      status: res.status,
      error: new Error(`Could not load ${url}`)
    }
  }
</script>

<script>
  import Gallery from 'svelte-brick-gallery'
  export let images
</script>

<Gallery {images} gap={0} itemHeight={340} />
```

Here is the link to the [two](https://github.com/elibenton/sveltekit-mdx/blob/main/src/routes/photos.svelte) [files](https://github.com/elibenton/sveltekit-mdx/blob/main/src/routes/photos.json.js) at play.
Should I refactor my code to work like yours? And why does mine cause this bug?

Thank you for any lead you could provide!

Eli

**Comment by anotherempty** [comment]
Thanks for your appreciation Eli.  

Svelte Brick Gallery uses [`Image()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image) to control image loading which is available after the DOM is loaded.

I suggest you to do like so :

```svelte
<script>
  import { onMount } from 'svelte'
  import Gallery from 'svelte-brick-gallery'
  export let images
  let mounted = false
  
  onMount(() => {
    mounted = true
  })
</script>

{#if mounted}
  <Gallery {images} gap={0} itemHeight={340} />
{/if}
```

**Comment by anotherempty** [comment]
I released version 0.0.10 that corrects this error so you don't have to check if the DOM is ready or not.

**Comment by elibenton** [comment]
Thank you so much! I appreciate both the quick fix and update!

On May 26, 2022, GitHub ***@***.***> wrote:
> I released version 0.0.10 that corrects this error so you don't have
> to check if the DOM is ready or not.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/anotherempty/svelte-brick-
gallery/issues/1#issuecomment-1138360957>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AESWRAV5SRMDZB73OBY44QLVL5CZPANCNFSM5WXSFOYQ>.
You are receiving this because you authored the thread.Message ID:
***@***.***>
